src/shipit_code_coverage/shipit_code_coverage/report_generators.py

```diff
@@ -37,21 +37,20 @@ def zero_coverage(artifacts, out_dir='code-coverage-reports'):
                 all_functions_uncovered = False
 
         if all_lines_uncovered or (len(sf['functions']) > 1 and all_functions_uncovered):
-            zero_coverage_files.append(name)
-
-    with open(os.path.join(out_dir, 'zero_coverage_files.json'), 'w') as f:
-        json.dump(zero_coverage_files, f)
+            zero_coverage_files.add(name)
 
     mkdir(os.path.join(out_dir, 'zero_coverage_functions'))
 
-    zero_coverage_function_counts = []
+    zero_coverage_report_counts = []
```

Comment:
Let's rename this to `zero_coverage_report`, since it now doesn't only contain function counts.